lua/harpoon/ui.lua

```diff
@@ -106,7 +112,13 @@ M.nav_file = function(id)
 
     vim.api.nvim_set_current_buf(buf_id)
     if set_row and mark.row then
-        vim.cmd(string.format(":call cursor(%d, %d)", mark.row, mark.col))
+        local ok, err = pcall(vim.cmd, string.format(":call cursor(%d, %d)", mark.row, mark.col))
```

Comment:
can this error?

Reply:
technically it can if string.format doesn't receive numbers, but I'm not actually sure if we would run into this. I'll change it around anyway for now.